api/components/wrap.schema.json

```
@@ -0,0 +1,62 @@
+{
+    "$schema": "https://raw.githubusercontent.com/lenra-io/ex_component_schema/beta/priv/static/draft-lenra.json",
+    "$id": "wrap.schema.json",
+    "title": "Wrap",
+    "description": "Element of type Wrap",
+    "type": "component",
+    "properties": {
+        "type": {
+            "description": "The identifier of the component",
+            "enum": [
+                "wrap"
+            ]
+        },
+        "children": {
+            "description": "The children",
+            "type": "array",
+            "items": {
+                "type": "component"
+            }
+        },
+        "direction": {
+            "$ref": "defs/direction.schema.json"
+        },
+        "crossAxisAlignment": {
+            "description": "The alignment along the cross axis",
+            "type": "string",
+            "enum": [
+                "start",
+                "end",
+                "center"
+            ],
+            "default": "start"
+        },
+        "spacing": {
+            "description": "The multiplier of the base size for the minimal spacing",
+            "type": "number",
+            "default": 0
+        },
+        "runSpacing": {
+            "description": "The multiplier of the base size for spacing between different line in Wrap",
```

```suggestion
            "description": "The multiplier of the base size for spacing between different runs in Wrap",
```
Not `line` because it can be both vertical or horizontal.